core/src/components/checkbox/test/label/checkbox.e2e.ts

```diff
@@ -138,5 +138,31 @@ configs().forEach(({ title, screenshot, config }) => {
         expect(await checkbox.screenshot()).toMatchSnapshot(screenshot(`checkbox-label-fixed-justify-space-between`));
       });
     });
+
+    test.describe('checkbox: stacked placement', () => {
+      test('should render a start alignment with label in the stacked position', async ({ page }) => {
```

Comment:
~ The test descriptions are a bit hard to decipher/read. 

Thoughts on rewording to:
> should align the label to the start of the container in the stacked position

This would apply to the center align test as well.